reservation_units/tests/factories.py

```diff
@@ -40,7 +41,7 @@ class TaxPercentageFactory(DjangoModelFactory):
     class Meta:
         model = "reservation_units.TaxPercentage"
 
-    value = FuzzyChoice(choices=(10.0, 14.0, 24.0))
+    value = FuzzyChoice(choices=(Decimal(10.0), Decimal(14.0), Decimal(24.0)))
```

Comment:
```suggestion
    value = FuzzyChoice(choices=(Decimal("10.0"), Decimal("14.0"), Decimal("24.0")))
```

I guess these are fine as is, but suggesting just for the consistency 😄